cernopendata/modules/fixtures/data/records/jade-computing-notes.json

```diff
@@ -0,0 +1,5267 @@
+[
+  {
+    "abstract": {
+      "description": "Index"
+    },
+    "authors": "",
+    "accelerator": "DESY-PETRA",
+    "collaboration": {
+      "name": "JADE collaboration",
+      "recid": "451"
+    },
+    "collections": [
+      "JADE-Computing-Notes"
+    ],
+    "date_created": [
+      ""
+    ],
+    "date_published": "2023",
+    "distribution": {
+      "formats": [
+        "pdf"
+      ],
+      "number_files": 1,
+      "size": 137529
+    },
+    "experiment": "JADE",
+    "files": [
+      {
+        "checksum": "adler32:46c1eb5f",
+        "size": 137529,
+        "uri": "root://eospublic.cern.ch///eos/opendata/jade/documentation/notes/0Index.pdf"
+      }
+    ],
+    "license": {
+      "attribution": "CC0"
+    },
+    "publisher": "CERN Open Data Portal",
+    "recid": "261300",
+    "title": "JADE Computing Note 0",
+    "type": {
+      "primary": "Supplementaries",
+      "secondary": [
+        "Computing Note"
+      ]
+    }
+  },
+  {
+    "abstract": {
+      "description": "Monte Carlo Formats"
+    },
+    "authors": "",
+    "accelerator": "DESY-PETRA",
+    "collaboration": {
+      "name": "JADE collaboration",
+      "recid": "451"
+    },
+    "collections": [
+      "JADE-Computing-Notes"
+    ],
+    "date_created": [
+      "Nov 09 1978"
```

Comment:
`date_created` is represented collision-taking year, please see the field description in `cernopendata/jsonschemas/records/record-v1.0.0.json`.  It must be set to "1978" only, so that facets etc would work. Please remove the month and date.

(If we need month and date, then we should use 1978-11-09" (i.e. YYYYMMDD) as we do in "date_published" fields for the documents, but then may need to amend facet handling to created proper year. Please let me know if you wish to store also months and dates.)